docs/DeveloperGuide.md

```diff
@@ -421,6 +421,20 @@ when editing the tags of the member/event:
 3. Edit current list of tags attached to the member/event
 4. Delete specific tags from the current list of tags attached to the member/event
 
+### Provide more specific index error messages to the user
+**Current Implementation**
```

Comment:
Should we add a colon here to be consistent with the other sub-headings?
```suggestion
**Current Implementation:**
```

Reply:
Nit picking here, but there are some inconsistencies in whether the colon is bolded or not in other subheadings as well, maybe we should just stick to bolding the colon?

Reply:
Okay, let's bold all colons!